Dockerfile

```diff
@@ -57,11 +57,12 @@ RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor -ldflags="-X 'main.Version=$KC
 # --- run image ---
 FROM photon:4.0
 
-RUN tdnf install -y git openssh-clients shadow-tools sed
+RUN tdnf install -y git openssh-clients sed
 
-RUN groupadd -g 2000 kapp-controller && useradd -r -u 1000 --create-home -g kapp-controller kapp-controller
-RUN chmod g+w /etc/pki/tls/certs/ca-bundle.crt && chgrp kapp-controller /etc/pki/tls/certs/ca-bundle.crt
-USER kapp-controller
+RUN echo "kapp-controller:x:1000:" > /etc/group && \
+    echo "kapp-controller:x:1000:1000::/home/kapp-controller:/usr/sbin/nologin" > /etc/passwd
+
+RUN chmod a+w /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
This sets the ca-bundle to world-writeable, so that no matter which user we are running as we'll be able to append certificates to it. I think ideally we would use [SSL_CERT_DIR](https://www.jvt.me/posts/2019/12/04/openssl-certs-dir-setup/) with an emptyDir volume, but that requires more work than I wanted to do here.

Reply:
Is there any sort of practical consequence of making the ca-bundle writable to all users?

Reply:
Yeah, this is what I'm most uncertain about in this change. I couldn't think of a way to scope the permissions to a specific user/group.